kodev

```diff
@@ -94,6 +94,10 @@ ${SUPPORTED_TARGETS}"
             assert_ret_zero $?
             ;;
         android)
+            if [ "$NDK" == "" ]; then
+                # Some distributions use `ANDROID_NDK` instead, fall back to it
```

Comment:
I meant `[ ${ANDROID_NDK+x} ]` (no `-z`, sorry for the typo) but it doesn't matter either way. It just feels inelegant to do the same check twice.